gulpfile.js/lib/webpack-multi-config.js

```diff
@@ -9,11 +9,15 @@ module.exports = function(env) {
   var jsSrc = path.resolve(config.root.src, config.tasks.js.src)
   var jsDest = path.resolve(config.root.dest, config.tasks.js.dest)
   var publicPath = path.join(config.tasks.js.dest, '/')
-  var filenamePattern = env === 'production' ? '[name]-[hash].js' : '[name].js'
+  var filenamePattern = '[name].js'
   var extensions = config.tasks.js.extensions.map(function(extension) {
     return '.' + extension
   })
 
+  if(config.tasks.production.rev.enable === true && env === 'production') {
+    var filenamePattern = '[name]-[hash].js'
```

Comment:
Good catch on both counts. The original commit had an `enable` property. This is a mistake. And yeah, that `var` needs to be killed. Yay code reviews!